src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigatorContext.java

```diff
@@ -69,6 +76,21 @@ public String getTeamSlug() {
         return teamSlug;
     }
 
+    /**
+     * Sets the topics which the repositories must have.
+     */
+    void setTopics(ArrayList<String> topics) {
```

Comment:
might fix your issue
```suggestion
    public void setTopics(ArrayList<String> topics) {
```

Reply:
I just tested it and unfortunately, this change didn't help.
I probably don't understand enough on how the interaction between front- and backend works here... However, one can't even see the topics when just hitting `save` and then going back to `Configure`. The changes take effect though :thinking:

Reply:
It's fixed in commit 81bde9b